dist/index.html

```diff
@@ -5,7 +5,7 @@
   <meta charset="utf-8">
   <title>Skeleton Framework Test File</title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link href="//fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
+  <link href="http://fonts.googleapis.com/css?family=Raleway:400,300,600" rel="stylesheet" type="text/css">
```

Comment:
Are you running `npm start` to fire up the server? If live-serve isn't working please submit a new issue. Any build/development issues can be resolved in a different commit.